tests/integration/helpers.py

```diff
@@ -51,16 +51,35 @@ def bootstrap_microceph() -> Tuple[CephFSShareInfo, CephFSAuthInfo]:
     _exec_commands(
         instance,
         [
-            ["ln", "-s", "/bin/true", "/usr/local/bin/udevadm"],
-            ["apt-get", "-y", "update"],
-            ["apt-get", "-y", "install", "ceph-common"],
-            ["snap", "install", "microceph"],
-            ["microceph", "cluster", "bootstrap"],
-            ["microceph", "disk", "add", "loop,2G,3"],
+            "ln -s /bin/true".split(),
+            "apt update -y".split(),
+            "apt upgrade -y".split(),
+            "apt install -y ceph-common".split(),
```

Comment:
Why no use lists directly here? Since we're not entering any dynamic string content, we could save ourselves any itybitty amount of time by using lists directly rather than splitting based on whitespace:

```suggestion
            ["ln", "-s", "/bin/true"],
            ["apt", "update", "-y"],
            ["apt", "upgrade", "-y"],
            ["apt", "install", "-y", "ceph-common"],
```

MUST BE BLAZINGLY FAST :rocket::rocket:!!